modules/distr-tracing-tempo-install-overview.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_content-type: CONCEPT
+[id="distr-tracing-tempo-install-overview_{context}"]
+= {DTProductName} installation overview
+
+The steps for installing {DTProductName} are as follows:
+
+* Review the documentation and determine your deployment strategy.
+
+* If your deployment strategy requires persistent storage, install the OpenShift Elasticsearch Operator via the OperatorHub.
```

Comment:
`OpenShift Elasticsearch Operator` is not required anymore for Tempo. We can copy [this sentence from Loki](https://docs.openshift.com/container-platform/4.10/networking/network_observability/installing-operators.html#network-observability-loki-installation_network_observability):
```
* Supported Log Store (AWS S3, Google Cloud Storage, Azure, Minio, OpenShift Data Foundation)
```

Note that Swift is currently not supported for the Tempo Operator.